application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,16 +46,32 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
-# Live table keys
+##Most Viewed Pages Table Header
+analytics.mostViewedPages.year=Year
+analytics.mostViewedPages.label=Page Title
+analytics.mostViewedPages.nb_visits=Unique Vistators
+analytics.mostViewedPages.nb_hits=Page Views
+analytics.mostViewedPages.sum_time_spent=Time spent(s)
+analytics.mostViewedPages.entry_nb_visits=No. Of Views That Started On This Page
+analytics.mostViewedPages.entry_nb_actions= No. Of Actions
+analytics.mostViewedPages.bounce_rate=Bounce Rate
+analytics.mostViewedPages.exit_rate=Exit Rate
```

Comment:
I think you should use `analytics.mostViewedPages.header` as prefix. I am never sure for translation keys, but it seems closer to the `module[.submodule]*[.section]*.element[.part]*` rule
Also, `Individual parts of the key should use camelCase if more than one word is needed to identify the element.`
For the sufix of the key, some might need an improvement IMO. Suggestion:
```
analytics.mostViewedPages.header.bounceRate=Bounce rate
analytics.mostViewedPages.header.entryNbVisits=No. of views that started on this page
analytics.mostViewedPages.header.entryNbActions= No. of actions
analytics.mostViewedPages.header.exitRate=Exit rate
analytics.mostViewedPages.header.hits=Page views
analytics.mostViewedPages.header.timeSpent=Time spent
analytics.mostViewedPages.header.title=Page title
analytics.mostViewedPages.header.visits=Unique visitors
analytics.mostViewedPages.header.year=Year
```